web/templates/popovers/color_picker_popover.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="popover-menu color-picker-popover" data-simplebar data-simplebar-tab-index="-1">
+    <div class="message_header message_header_stream" data-stream-id="{{stream_id}}">
+        <div class="message-header-contents" style="background: {{recipient_bar_color}};">
+            <div class="message_label_clickable stream_label">
+                <span class="stream-privacy-modified-color-{{stream_id}} stream-privacy filter-icon"  style="color: {{stream_privacy_icon_color}}">
+                    {{> ../stream_privacy .}}
+                </span>
+                {{#if stream_name}}
+                <span class="message-header-stream-name">
+                    {{~stream_name~}}
+                </span>
+                {{else}}
+                &nbsp;
+                {{/if}}
+            </div>
+        </div>
+    </div>
+    <ul role="menu" class="popover-menu-list">
+        <li role="none">
+            <div role="group" class="color-swatch-list" aria-label="{{t 'App theme' }}">
```

Comment:
Why is this "App theme" as the label for screenreaders?

Reply:
Fixed. Sorry, I copied the theme switcher radio button group structure as the base for the color swatches, and missed these labels.